UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/FileContentHelper.cs

```diff
@@ -91,5 +91,23 @@ public static async Task<List<string>> CreateExchangeSetFileForIsoAndSha1Files(s
             }
             return downloadFolderPath;
         }
+        public static async Task<List<string>> DownloadCatalogueXmlOrEncUpdatesListCsvFileForLargeMedia(string batchId, string fssJwtToken, dynamic batchDetailsResponse)
```

Comment:
Duplication of code. This function can be simplified as follows:

            string responseContent = batchDetailsResponse.attributes[5].value;

            string downloadFileUrl = $"{Config.FssConfig.BaseUrl}/batch/{batchId}/files/";
            downloadFileUrl += responseContent.Equals("Catalogue")
                ? posDetails.AVCSCatalogueFileName
                : posDetails.EncUpdateListFileName;

            string downloadFile = await FssBatchHelper.DownloadFileForLargeMedia(downloadFileUrl, fssJwtToken);

            List<string> downloadFolderPath = new()
            {
                downloadFile
            };

            return downloadFolderPath;